docs/experiments/extract-mode.md

```diff
@@ -66,7 +66,7 @@ else:
     print("Success")
 ```
 
-Likewise, if our query included a Dockerfile with an equivalent Bash command, such as the example below, Semgrep would report the contained Bash as matching against the `curl-eval` rule.
+Likewise, if a query included Dockerfile with an equivalent Bash command, Semgrep reports the contained Bash as matching against the `curl-eval` rule. See the following Dockerfile example that contains a Bash command:
```

Comment:
Most of this looks good, but I believe it should still be "a Dockerfile". To the best of my knowledge, Docker doesn't call the "language" Dockerfiles are written in anything specific, and just refers to it as a text format, so I believe this ought to refer to a specific file. See, e.g., the usage https://docs.docker.com/engine/reference/builder/.

Reply:
Good to know, thanks! Changing to "a Dockerfile".